src/docs/user_ai.md

```diff
@@ -72,6 +73,16 @@ As an alternative to using an official OpenAI account, Theia IDE also supports a
 }
 ```
 
+### Anthropic
+
+To enable Anthropics AI models in the Theia IDE, create an API key in your Anthropics account and
+enter it in the Theia IDE settings under AI-features => Anthropics.
+**Please note:** The Anthropics API key will be stored in clear text. Use the environment variable `ANTHROPIC_API_KEY` to set the key securely.
```

Comment:
```suggestion
To enable Anthropics AI models in the Theia IDE, create an API key in your Anthropics account and
enter it in the Theia IDE settings under AI-features => Anthropics.

**Please note:** The Anthropics API key will be stored in clear text. Use the environment variable `ANTHROPIC_API_KEY` to set the key securely.
```

![image](https://github.com/user-attachments/assets/6c1395fa-6c29-4a82-bd49-1d9113fb4515)

I feel like this looks a bit crowded on the rendered page. Should we maybe add a newline here so that it is displayed as a separate block? (Same is true for the huggingface section.